### DISCUSSION TITLE
Restore callbacks states

### DISCUSSION THREAD

**Discussion by ghost** [Lightning-AI/pytorch-lightning#10284] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
https://pytorch-lightning.readthedocs.io/en/latest/common/weights_loading.html#restoring-training-state

In this last example, how to access callbacks after restoration? Is there something like, restored_trainer.restored_callback_states? I'm asking because the best_path_model can only be accessed if we have access to the model_checkpoint_callback.

**Comment by tchaton** [comment]
You could do `trainer.callbacks` to access all callbacks.
